docs-source/content/faq/boot-identity-not-valid.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Boot Identity Not Valid"
+date: 2020-03-02T08:08:19-04:00
+draft: false
+weight: 1
```

Comment:
Minor complaint:  There are already two other FAQ entries at weight 1. It'd be nice if the ordering of FAQ entries was more predictable.